### DISCUSSION TITLE
Install directly from GitHub using PIP

### DISCUSSION THREAD

**Discussion by dipu-bd** [lncrawl/lightnovel-crawler#1674] [Show and tell]
The `master` branch contains the latest stable code. If you can not wait for it to be released in the PyPi, you can get it like this:

```
$ pip install -U git+https://github.com/dipu-bd/lightnovel-crawler.git#egg=lightnovel-crawler
```

The `dev` branch contains cutting-edge, sometimes unstable changes. To install it:

```
$ pip install -U https://github.com/dipu-bd/lightnovel-crawler/tarball/refs/heads/dev#egg=lightnovel-crawler
```

**Comment by budikesuma** [comment]
@dipu-bd 

If I want to go back to a certain version(v2.34.0) through the source code, can this command be executed directly?

```
pip install -U https://github.com/dipu-bd/lightnovel-crawler/archive/refs/tags/v2.34.0.zip
```

Or this 2nd will be better? 

```
pip uninstall lightnovel-crawler && pip install -U https://github.com/dipu-bd/lightnovel-crawler/archive/refs/tags/v2.34.0.zip
```

**Reply by dipu-bd** [reply]
why not `pip install lightnovel-crawler==2.34.0` ?
